src/index.html

```diff
@@ -1,14 +1,43 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="linthtml">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header-content">
+        <div class="logo">
+          <a class="nav-link" href="#home"
+            ><img src="images/logo.png" alt="Moyo logo"></a>
+        </div>
+        <nav class="nav-list">
+          <ul class="navigation">
+            <li class="nav-link is-active"><a href="#apple">Apple</a></li>
+            <li class="nav-link"><a href="#samsung">Samsung</a></li>
```

Comment:
There will be better readability with tags on separate lines
```suggestion
            <li class="nav-link">
               <a href="#samsung">Samsung</a>
            </li>
```